micropipenv.py

```diff
@@ -971,9 +975,21 @@ def _get_package_entry_str(
             result += "@{}".format(info["ref"])
         result += "#egg={}\n".format(package_name)
         return result
+    if "path" in info:
+        # Path formats we want to support:
+        # - "file:///path/to/project"
+        # - "/path/to/project"
+        # - "./path/to/project"
+        # - "project" (== "./project")
+        # - "."
+        if "/" not in info["path"] and not info["path"].startswith("."):
+            # Assume a relative path
+            info["path"] = "./{}".format(info["path"])
```

Comment:
Not sure if I read this correctly - would this work even if the path is a directory starting with a dot - e.g. `.mylib`? Probably not that common, but just to clarify.

Reply:
That, from my experience, depends on pip. Whatever is a valid package name, pip tries to find it in the index. But if it contains "/", it's no longer a valid name so it tries to find it locally. It seems that dot is not allowed either so ".testproject" should work.

Reply:
Ah, the result is passed to pip as an argument. I missed the bigger picture here. Thanks for the clarification. 👍🏻